ansible_collections/arista/avd/docs/getting-started/avd-aap.md

```diff
@@ -0,0 +1,403 @@
+
+- An accessible lab topology running Arista EOS.
+- An AVD project or Git repository with playbooks and an inventory. To get started, you may also use any of our [example topologies](../../examples/single-dc-l3ls/README.md).
+- A RHEL instance running AAP.
+  - If you need access to a RHEL instance, you can join the [developer program](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux/developer-program) to get a copy.
+  - To get started, you may also sign up for a 60-day [trial license](https://www.redhat.com/en/technologies/management/ansible/trial) for AAP.
+
+## Topology
+
+Our topology leverages two spines and four leaf nodes to create a layer three leaf spine topology. The topology is managed by Arista CloudVision (CV). In our case, the EOS nodes are constantly streaming to CV and gives us the ability to provision the nodes with CV. AAP will act as our controller to communicate any updates to CV, which can then push the updates to our topology.
+
+![Topology leveraging Ansible Automation Platform to communicate with CloudVision](../_media/getting-started/aap-avd/aap-cvp-eos.svg)
+
+## AAP Dashboard
+
+The image below provides an excellent overview of the AAP dashboard. From one pane, we have quick links to hosts, inventories, and breakdowns of recent job executions. The left pane provides us access to anything we may need within AAP. This guide will go through setting up the following items:
+
+- Execution environments
+- Projects
+- Inventories
+- Job templates
+- Surveys
+
+![AAP Dashboard](../_media/getting-started/aap-avd/aap-dashboardpng.png)
+
+## Execution environments with Ansible Builder
+
+Execution environments (EEs) are Red Hat's solution for managing project dependencies. In the past, we used Python virtual environments. The Python virtual environments had pros and cons, but EEs leverage containers to wrap all dependencies within a container. EEs make environments more portable and quicker to replicate between AAP nodes and standalone projects. We will go through an EE build together but it is highly recommended to build your own for your specific requirements.
+
+### Ansible Builder
+
+Ansible Builder is a tool developed by the Ansible team to aid in creating EEs. Like any tool, there are a few requirements to get started with Ansible Builder.
+
+- A development machine with Python installed.
+- Ansible builder installed with `pip install ansible-builder`.
+- [Docker](https://docs.docker.com/engine/install/) or [Podman](https://podman.io/docs/installation) installed on the development machine.
+
+You can place the Ansible builder dependencies with your current project or leverage a separate project entirely for your EE builds. Please note that this example was created with version 4.7.1 of the AVD collection. Dependencies may change between versions.
```

Comment:
Can be a good idea to test with 4.9.0-dev. Just to be sure it works in the same way.

Reply:
docs have been updated to AVD 4.9.0 release.